app/setup/index.js

```diff
@@ -0,0 +1,259 @@
+import fetch from 'isomorphic-fetch';
```

Comment:
is there a more descriptive name than `setup/index.js`? like `login/index.js`? `setup` sounds like it could be used just for development setup